src/xunit.assert/xunit.assert.csproj

```diff
@@ -3,6 +3,11 @@
   <PropertyGroup>
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
     <TargetFramework>netstandard2.0</TargetFramework>
+    <LangVersion>8</LangVersion>
```

Comment:
To be pedantic, `8` is not in the [list of supported langVersions](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/compiler-options/langversion-compiler-option).

Reply:
Thanks. It certainly works. I've asked internally whether this is a deficiency in the docs or something that should be changed in the project.

Reply:
The conclusion is that either is fine. https://github.com/dotnet/roslyn/blob/master/src/Compilers/CSharp/Portable/LanguageVersion.cs#L340